### ISSUE TITLE
2nd GPU wont load

### ISSUE THREAD

**Issue by KooSkwan** [nanopool/ewbf-miner#16]
Hey guys im having trouble getting my 2nd GPU to hash, the card reads on my computer so its there here tried using --solver 0 1 2 3 as well as --cuda_devices 0 1 2 3. what am i doing wrong here??

**Comment by ibmua** [comment]
`--cuda_devices 0 1` works for me

**Comment by JQE** [comment]
Are the drivers installed and showing in device manager? Have you rebooted
since installing the card drivers?

**Comment by KooSkwan** [comment]
Yes both cards are in device manager and they have been restarted along with the drivers

**Comment by KooSkwan** [comment]
i tried --cuda_devices 0 1 still nothing... =[

**Comment by KooSkwan** [comment]
i am using miner.exe --server us1-zcash.flypool.org --user t1ZuSpEM6T871T2CMWnacUz4E9xn5jDvYvN.PC1 --pass x --port 3333 --cuda_devices 0 1 --pec not sure why is not working....

**Comment by ChriscomIT** [comment]
What happens if you try to start without the "cuda_devices" parameter. If I remember correctly it then should automatically try to start mining on all detected GPUs.

**Comment by KooSkwan** [comment]
dude... i tried that earlier and it did not work but now it is. thank you for your help guys.... happy mining!

**Comment by ChriscomIT** [comment]
You are welcome! Happy mining!
